api/v1/swagger.yaml

```diff
@@ -8028,6 +8028,240 @@ paths:
           $ref: "#/responses/403"
         default:
           $ref: "#/responses/genericError"
+# =============================================================================
+  /botActions:
+    x-swagger-router-controller: botActions
+    get:
+      summary: Retrieve all botActions
+      tags: [ botActions ]
+      operationId: findBotActions
+      parameters:
+        -
+          name: name
+          description: Get botActions by name
+          in: query
+          required: false
+          type: string
+        -
+          name: isPending
+          description: Get pending botActions
+          in: query
+          required: false
+          type: boolean
+        -
+          $ref: "#/parameters/limitParam"
+        -
+          $ref: "#/parameters/offsetParam"
+      responses:
+        200:
+          description: >-
+            Success, returns all BotActions
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/BotActionResponse"
+        400:
+          $ref: "#/responses/400"
+        401:
+          $ref: "#/responses/401"
+        404:
+          $ref: "#/responses/404"
+        default:
+          $ref: "#/responses/genericError"
+
+    post:
+      security:
+        - jwt: []
+      summary: Create a new botAction
+      tags: [ botActions ]
+      description: Create a new botAction with a name, room id and bot id
+      operationId: postBotActions
+      parameters:
+        -
+          name: queryBody
+          in: body
+          required: true
+          schema:
+            type: object
+            description: Create botAction properties
+            properties:
+              name:
+                type: string
+                maxLength: 254
```

Comment:
The name validation is missing here too. If this is something that you just started adding, we can ignore it for now and you can take care of it in the next pr. Let me know.